packages/frontend/src/admin-pages/components/forms/Ckeditor4.vue

```diff
@@ -47,6 +47,7 @@ export default {
       default: 'basic'
     }
   },
+  emits: ['input'],
```

Comment:
Ckeditor4 does not appear to be working.  Should we remove it - it appears to be for old legacy support?  Probably test v5 with Edge and remove v4 if it isn't needed. CKeditor.vue has:
```
    editor () {
      const vm = this
      return vm.isInternetExplorer() || vm.isMicrosoftEdge() ? 'forms-ckeditor4' : 'forms-ckeditor5'
    }
```